deepmd/pt/model/model/spin_model.py

```
@@ -469,6 +445,7 @@ def forward_common_lower(
         fparam: Optional[torch.Tensor] = None,
         aparam: Optional[torch.Tensor] = None,
         do_atomic_virial: bool = False,
+        comm_dict: Optional[Dict[str, torch.Tensor]] = None,
```

Ruff

Undefined name `Dict`

(F821)